Dockerfile

```
@@ -1,14 +1,22 @@
-FROM golang:1.17.5-alpine3.15 as builder
-RUN apk add --no-cache git ca-certificates make tzdata
-COPY . /app
-RUN cd /app && \
-    go get -d -v && \
-    CGO_ENABLED=0 GOOS=linux go build -v -a -installsuffix cgo -o prometheus_bot
+FROM golang:1.17.6-alpine3.15 as builder
+RUN addgroup -S appgroup && adduser -S appuser -G appgroup -u 1000 \
+  && apk add --update --no-cache ca-certificates tzdata
```

There is no need to specify `--update` when `--no-cache` is present.
```suggestion
  && apk add --no-cache ca-certificates tzdata
```

---

Actually seems `--update` depricated, I can't find it via `apk add -h`
Good catch, thank you.